pkg/runtime/wfengine/wfengine.go

```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
 }
 
 var (
+	WorkflowActorType   = actors.InternalActorTypePrefix + "wfengine.workflow"
+	ActivityActorType   = actors.InternalActorTypePrefix + "wfengine.activity"
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID string
+}
+
+func getNamespace() string {
+	namespace := os.Getenv("NAMESPACE")
+	if namespace == "" {
+		namespace = defaultNamespace
+	}
+	return namespace
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	WorkflowActorType = WorkflowActorType + "." + getNamespace() + "." + config.AppID
+	ActivityActorType = ActivityActorType + "." + getNamespace() + "." + config.AppID
```

Comment:
ideally should be `ns.appID.<var>` namespace should be a prefix?

Reply:
Please add UTs for the same...

Reply:
> ideally should be `ns.appID.<var>`. `namespace` should be a prefix?

Actually, the purpose is just to differentiate two Apps and if namespace available, then specifying that too, for better differentiation and recognition by underlying WF engine. 
Right now, many entries are formed in state Stores for a Workflow and Activities, with Internal Actor having representation like:

dapr.internal.wfengine.workflow.default.wfapp

dapr.internal.wfengine.activity.default.wfapp

I feel it would just elongate the Actor type naming, with `namespace` and `appId`, without serving any actual purpose.
WDYT?

Reply:
> Please add UTs for the same...

Now, asserting WorkflowActorType and ActivityActorType in `TestStartWorkflowEngine`.

Reply:
@DeepanshuA I think Mukandan is just suggesting a rearrangement of the existing components, and not asking that we literally add "namespace" and "appId". How about if we want to rearrange to something like `dapr.internal.mynamespace.myapp.workflow` and `dapr.internal.mynamespace.myapp.activity`? That might make for a slightly cleaner hierarchy.

Reply:
yes that is what I meant.